app/src/main/java/org/tuerantuer/launcher/ui/screen/AllAppsScreen.kt

```diff
@@ -30,25 +49,78 @@ fun AllAppsScreen(
     onOpenApp: (appItemInfo: AppItemInfo) -> Unit = {},
     onGoBack: () -> Unit = {},
 ) {
+    val searchQuery = remember { mutableStateOf("") }
+    val filteredList = remember { mutableStateOf(uiState.allApps) }
```

Comment:
wenn man jetzt ne neue App installiert wird und sich uiState.allApps ändert, wird filteredList nicht aktualisiert und die neue App erscheint nicht in der App-Liste. Würde deshalb noch den check einbauen ob sich die allApps geändert hat und wenn ja, filteredList nochmal neu berechnen (kannst du aus wg Performance auch den !== check nehmen anstatt !=)